pkg/kubelet/kubelet.go

```diff
@@ -1521,6 +1538,10 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		return fmt.Errorf("no node instance returned for %q", kl.hostname)
 	}
 
+	if err := kl.reconcileCbr0(node.Spec.PodCIDR); err != nil {
```

Comment:
Updated the function comment. @vmarmol, you know better than I do about the intended structure of the kubelet, so feel free to suggest alternatives.